walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,110 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.lang.Math;
 
 public final class FindMeetingQuery {
```

Comment:
Add a javadoc here about what the class does (maybe copy the description from the PR?)

Reply:
Done.